src/libraries/Native/Unix/CMakeLists.txt

```diff
@@ -195,15 +195,19 @@ endif()
 
 add_subdirectory(System.Native)
 
-if (NOT CLR_CMAKE_TARGET_ARCH_WASM AND NOT CLR_CMAKE_TARGET_IOS)  # TODO: reenable for iOS
+if (NOT CLR_CMAKE_TARGET_ARCH_WASM)
+    add_subdirectory(System.Net.Security.Native)
+
+if (NOT CLR_CMAKE_TARGET_IOS)
```

Comment:
nit: indent `if` block with 4-space to match rest of the file.

Reply:
yup. I don't think I want to reset CI for that so I'll do it in my other PR :)